waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -284,3 +287,38 @@ proc connectToNodes*(pm: PeerManager,
   # This issue was known to Dmitiry on nim-libp2p and may be resolvable
   # later.
   await sleepAsync(chronos.seconds(5))
+
+# Ensures a healthy amount of connected relay peers
+proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
```

Comment:
Functions usually have "verb" names, e.g., `**get**ConnectedPeersById`. This function name has a "noun" name. Could you rename it? 

For example: if it spawns a coroutine that contains a loop, I would rename it to `spawnRelayConnectivityLoop()`.